keyboards/crkbd/keymaps/madhatter/config.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#define EE_HANDS
+
+#define SSD1306OLED
```

Comment:
Would prefer to move off the older SSD1306 implementation, and move to the OLED Driver feature. 

It doesn't need to be done now, but it is something to think about.

Reply:
Thanks. I'll hold off on these changes for now, since I don't actually plan to use the OLED anyway.